proposals/Level-1.md

```diff
@@ -93,19 +92,19 @@ Each exception type has a `type signature`. The type signature defines the list
 of values associated with the exception.
 
 Within the module, exception types are identified by an index into the
-[exception index space](#exception-index-space). This index is referred to as
-the `exception tag`. The `tagged exception type` is the corresponding
+[exception index space](#exception-index-space). This static index refers to the
+corresponding runtime index that uniquely identifies the exception type, and is
+called the `exception tag`. The `tagged exception type` is the corresponding
```

Comment:
Terminology nitpick: Technically what's defined in the exception section is not a type. What an exception definition defines is a fresh tag, which is a runtime *value* that *has* the ascribed exception type (and multiple tags can have the same type yet be different tags). This may be nitpicking, but the difference is that it clarifies that (1) multiple tags can have the same type, and (2) dispatch is on values not types (types have no runtime representation in Wasm).

Reply:
Ok. reworking this text.